src/app/services/nearbyCities.service.js

```diff
@@ -0,0 +1,48 @@
+export default class NearbyCitiesService {
+    constructor($http) {
+        this.$http = $http;
+    }
+
+    getNearbyCities(city) {
+        const apiKey = process.env.API_KEY;
+        const request = {
+            method: 'GET',
+            url: `http://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${apiKey}`,
+            params: {
+                units: 'metric',
+                mode: 'json'
+            }
+        };
+        const self = this;
+        return this.$http(request)
+            .then(response => {
+                let lat = response.data.coord.lat;
+                let lon = response.data.coord.lon;
```

Comment:
can these variable be `const`?? if you use babel you can use destructuring object properties to do something like this
```
const { lat, lon } = response.data.coord;